### DISCUSSION TITLE
Discord Channel

### DISCUSSION THREAD

**Discussion by TranceDevCode** [rwf2/Rocket#2843] [General]
Hi guys, I'm learning Rust and I want to learn about Rocket Framework... 
Searching the Rocket website I didn't find a Discord channel, so I created one but I would like your permission to share it here, honestly I really liked the language and I would like to contribute here to the framework in some way, I prefer to post the link under your authorization and not start posting a channel that is not official of the framework but I would love to make it so.

Greetings and I look forward to your response friends of the Rocket Framework team 😎🫡

**Comment by wezm** [comment]
There's already a Matrix room for chat about Rocket: https://matrix.to/#/#rocket:mozilla.org it's also accessible via IRC: https://kiwiirc.com/client/irc.libera.chat/#rocket

**Comment by the10thWiz** [comment]
At this point in time, we are not interested in adding another chat room. As noted above, we have a Matrix room.
